recorder/src/main/java/com/focusit/jsflight/recorder/internalevent/filestrategy/NewFileStrategy.java

```diff
@@ -0,0 +1,32 @@
+package com.focusit.jsflight.recorder.internalevent.filestrategy;
+
+import java.io.File;
+
+import com.focusit.jsflight.recorder.internalevent.InternalEventRecorder;
+
+/**
+ * Created by dkolmogortsev on 06.03.17.
+ * Strategy for acquiring new output file
+ */
+public interface NewFileStrategy
+{
+    /**
+     * @return output path
+     */
+    String getConfiguredOutputPath();
+
+    /**
+     * @return new storage file
+     */
+    File getNew();
+
+    /**
+     * Check if record must be stored in a new File
+     * @param record to check
+     * @return true - to new file, otherwise - not
+     */
+    default boolean isToNewFile(InternalEventRecorder.InternalEventRecord record)
```

Comment:
Why you've provide a defoult method and "abstract" class? it looks like overhead or I'm missing something